ee/tasks/test/test_send_license_usage.py

```diff
@@ -46,6 +45,7 @@ def test_send_license_usage(self, mock_post, mock_capture):
             self.user.distinct_id,
             "send license usage data",
             {"date": "2021-10-09", "events_count": 3, "license_keys": ["enterprise"], "organization_name": "Test"},
+            groups={"instance": ANY, "organization": str(self.organization.id)},
```

Comment:
We could just check this statically to make sure it works as expected, I think this should be `localhost:8000`, the default for `SITE_URL`